images/win/scripts/Installers/Windows2019/Install-VS2019.ps1

```diff
@@ -73,6 +73,9 @@ $WorkLoads = '--allWorkloads --includeRecommended ' + `
               '--add Microsoft.Component.VC.Runtime.UCRTSDK ' + `
               '--add Microsoft.Net.ComponentGroup.4.6.2.DeveloperTools ' + `
               '--add Microsoft.Net.ComponentGroup.4.7.1.DeveloperTools ' + `
+              '--add Microsoft.Net.Component.4.7.2.SDK ' + `
```

Comment:
Do we really need `.SDK` and `. TargetingPack` here?
I wonder because for all other frameworks only `DeveloperTools` is installed

Reply:
SDK is required to develop applications for the .NET Framework and
Targeting pack is required for targeting a particular .NET framework which is different from currently installed .NET framework.
Considering that we add additional .NET framework, they are vital.